<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_filename">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>15</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_xRot_left">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>75</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_zRot">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_z_move_left">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>275</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>385</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Ребра</string>
    </property>
    <widget class="QRadioButton" name="radioButton_solid_edge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сплошная линия</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_dotted_edge">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пунктирная линия</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_color_edge">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать цвет</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Толщина</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_color_edge">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>64</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_thickness_edge">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_zRot_right">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>135</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_film_process">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>965</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_edge_amount">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>915</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>280</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertex_amount">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>945</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scale">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>318</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_z_move_right">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>275</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_image_bmp">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>785</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>bmp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x_move_left">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>215</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scale_name">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>322</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_yRot">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>140</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_y_move_left">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>245</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zRot_left">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>135</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>220</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_yRot_right">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>105</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_move">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_film_process_5">
    <property name="geometry">
     <rect>
      <x>1252</x>
      <y>965</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_y_move_right">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>245</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>765</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить изображение в формате:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply_rotate">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>49</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Крутим барабан</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y_move">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>250</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>191</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кнопки для перемещения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>55</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кнопки для поворота</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_background">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>671</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет фона</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>110</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scale">
    <property name="geometry">
     <rect>
      <x>1196</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_yRot_left">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>105</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>345</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Съешь меня</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_film">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>815</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Записать скринкаст</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>855</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Информация</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>705</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Проекция</string>
    </property>
    <widget class="QRadioButton" name="radioButton_central">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>23</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Центральная</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ortho">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>23</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Параллельная</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_color_background">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>665</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет фона</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>885</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_xRot_right">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>75</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>353</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>915</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество ребер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply_move">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>185</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Тудым сюдым</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x_move_right">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>215</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_image_jpeg">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>785</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>jpeg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>250</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>345</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выпей меня</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_xRot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>945</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество вершин</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>525</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Вершины</string>
    </property>
    <widget class="QRadioButton" name="radioButton_no_point">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Убрать</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_cube">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_color_vertex">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать цвет</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Размер</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_color_vertex">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>64</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_sphere">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Круг</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dot_size">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_z_move">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>280</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Error">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="glView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>941</width>
      <height>961</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QWidget</extends>
   <header location="global">View/glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
